extension/src/wasm/bergamot-translator-worker.appendix.js

```diff
@@ -0,0 +1,212 @@
+/* global addOnPreMain, Module */
+
+addOnPreMain(function() {
+  let model;
+
+  const log = (message) => {
+    postMessage({
+      type: "log",
+      message,
+    })
+  }
+
+  const loadModel = (from, to) => {
+    log(`loadModel(${from}, ${to})`);
+    let start = Date.now();
+
+    // Delete previous instance if another model was loaded
+    if (model) {
+      model.delete();
+    }
+
+    const languagePair = `${from}${to}`;
+
+    // Vocab files are re-used in both translation directions
+    const vocabLanguagePair = from === "en" ? `${to}${from}` : languagePair;
+
+    // Set the Model Configuration as YAML formatted string.
+    // For available configuration options, please check: https://marian-nmt.github.io/docs/cmd/marian-decoder/
+    // This example captures the most relevant options: model file, vocabulary files and shortlist file
+    const modelConfig = `models:
+  - /${languagePair}/model.${languagePair}.npz
+vocabs:
+  - /${vocabLanguagePair}/vocab.${vocabLanguagePair}.spm
+  - /${vocabLanguagePair}/vocab.${vocabLanguagePair}.spm
+beam-size: 1
+normalize: 1.0
+word-penalty: 0
+max-input-sentence-tokens: 128
+max-input-tokens: 1024
+workspace: 128
+max-length-factor: 2.0
+skip-cost: true
+cpu-threads: 1
+quiet: true
+quiet-translation: true
+shortlist:
+    - /${languagePair}/lex.${languagePair}.s2t
```

Comment:
This means your lexical shortlist file should also have language identifier in the name. Do you have it?

Reply:
Yes